setup.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+"""The setup script."""
+import importlib
+from setuptools import setup
+
+readme = ''
+changes = ''
+
+setup_requires = []
+install_requires = [
+    'requests==2.22.0',
+    'eth-abi==2.1.0',
+    'pysha3==1.0.2',
+    'ecdsa==0.15',
+    'secp256k1==0.13.2',
+    'protobuf==3.11.3'
+]
```

Comment:
FYI(this flag means you don't need to change it in this PR)
these requires should be moved to a config file named `requirements.txt` since setup.py is a tool.
We don't change a tool's code if there is a config file when we need to install or upgrade pip package.

FYI

Reply:
if you‘re using `requirements.txt` for keeping requirements ,and should remove these `install_requires` to DRY your codes.